api/test/integration/test_manager_endpoints.tavern.yaml

```diff
@@ -1581,35 +1593,11 @@ stages:
         error: 0
         data:
           last_check_date: !anystr
-          current_version: v4.8.0
+          current_version: "v{wazuh_version:s}"
           update_check: true
-          last_available_major:
-            tag: v5.0.0
-            description: ""
-            title: Wazuh 5.0.0
-            published_date: '2023-10-05T12:48:00Z'
-            semver:
-              major: 5
-              minor: 0
-              patch: 0
-          last_available_minor:
-            tag: v4.9.1
-            description: ""
-            title: Wazuh 4.9.1
-            published_date: '2023-10-05T12:47:00Z'
-            semver:
-              major: 4
-              minor: 9
-              patch: 1
-          last_available_patch:
-            tag: v4.8.2
-            description: ""
-            title: Wazuh 4.8.2
-            published_date: '2023-10-05T12:46:00Z'
-            semver:
-              major: 4
-              minor: 8
-              patch: 2
+          uuid: !anystr
+      verify_response_with:
+        - function: tavern_utils:validate_update_check_response
```

Comment:
I chose this way because we will point out these tests to a staging server, or something like that, of the CTI service instead of a mock. So,  I think that the condition to retrieve always the same data can't be provided, with this method can we get more flexibility.